src/main/java/com/codessquad/qna/web/QuestionController.java

```diff
@@ -1,39 +1,51 @@
 package com.codessquad.qna.web;
 
+import com.codessquad.qna.domain.Qna;
+import com.codessquad.qna.domain.QnaRepository;
+import com.codessquad.qna.exception.NoQuestionException;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
-
-import java.util.ArrayList;
-import java.util.List;
+import org.springframework.web.bind.annotation.RequestMapping;
 
 @Controller
+@RequestMapping("/qna")
 public class QuestionController {
+    private final QnaRepository qnaRepository;
+
+    public QuestionController(QnaRepository qnaRepository) {
+        this.qnaRepository = qnaRepository;
+    }
 
-    List<Qna> qnaList = new ArrayList<>();
+    @PostMapping
+    public String createNewQna(Qna qna) {
+        if(checkNull(qna)){
+            return "qna/form";
+        }
+        else {
+            qna.setCreatedDateTime(qna.getFormattedDateTime());
+            qnaRepository.save(qna);
+            return "redirect:/";
+        }
```

Comment:
```suggestion
        qna.setCreatedDateTime(qna.getFormattedDateTime());
        qnaRepository.save(qna);
        return "redirect:/";
```

early return을 했으면 else는 필요 없습니다.